### PR TITLE
Merge of backend's first version with main

### PR DESCRIPTION
**Pull request by Jacob-Michalski**
As requested, I make this pull request so you can start working on the pipeline

### CODE REVIEW

File: backend/src/main.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from typing import List
+from fastapi import FastAPI, Depends
+from sqlmodel import Session, select
+
+from .model import Artist, Album, Genre, Track
+from .db import init_db, get_session
+
+class Song():
```
Comment (Jeremie-OP) [inline backend/src/main.py]: why ?

Comment (Jacob-Michalski) [reply backend/src/main.py]: Because
I forgot to delete this utility class
Will be fixed on next commit

Comment (Jeremie-OP) [reply backend/src/main.py]: Ok, I'm waiting this commit before merge :)

Comment (Jacob-Michalski) [reply backend/src/main.py]: On it

Comment (Jeremie-OP) [reply backend/src/main.py]: Perfect

**Pull request merged**